config/initializers/logjam_agent.rb

```diff
@@ -1,6 +1,6 @@
 module LogjamAgent
   # Configure the application name (required). Must not contain dots or hyphens.
-  self.application_name = "canvas-lms"
+  self.application_name = "canvas_lms"
```

Comment:
`Must not contain dots or hyphens.` 
🤔I guess this was a necessary change...lol